chains/orchestrator-relays/runtime/starlight/constants/src/lib.rs

```diff
@@ -0,0 +1,190 @@
+    // <https://research.web3.foundation/en/latest/polkadot/BABE/Babe/#6-practical-results>
+    pub const PRIMARY_PROBABILITY: (u64, u64) = (1, 4);
+}
+
+pub mod snowbridge {
+    use {
+        frame_support::parameter_types,
+        xcm::prelude::{Location, NetworkId},
+    };
+
+    parameter_types! {
+            /// Network and location for the Ethereum chain. On Stagelight, the Ethereum chain bridged
+            /// to is the Sepolia Ethereum testnet, with chain ID 11155111.
+            /// <https://chainlist.org/chain/11155111>
+            /// <https://ethereum.org/en/developers/docs/apis/json-rpc/#net_version>
+            pub EthereumNetwork: NetworkId = NetworkId::Ethereum { chain_id: 11155111 };
+            pub EthereumLocation: Location = Location::new(1, EthereumNetwork::get());
+
+    }
+
+    #[cfg(feature = "runtime-benchmarks")]
+    parameter_types! {
+            // We need a different ethereum location for benchmarks as the ethereum system pallet
+            // is written for benchmarks from para
+            pub EthereumLocationForParaIdBenchmarks: Location = Location::new(2, EthereumNetwork::get());
+
+    }
+}
+
+/// Fee-related.
+pub mod fee {
+    pub use sp_runtime::Perbill;
+    use {
+        crate::weights::ExtrinsicBaseWeight,
+        frame_support::weights::{
+            WeightToFeeCoefficient, WeightToFeeCoefficients, WeightToFeePolynomial,
+        },
+        primitives::Balance,
+        smallvec::smallvec,
+    };
+
+    /// The block saturation level. Fees will be updates based on this value.
+    pub const TARGET_BLOCK_FULLNESS: Perbill = Perbill::from_percent(25);
+
+    /// Handles converting a weight scalar to a fee value, based on the scale and granularity of the
+    /// node's balance type.
+    ///
+    /// This should typically create a mapping between the following ranges:
+    ///   - [0, `frame_system::MaximumBlockWeight`]
+    ///   - [Balance::min, Balance::max]
+    ///
+    /// Yet, it can be used for any other sort of change to weight-fee. Some examples being:
+    ///   - Setting it to `0` will essentially disable the weight fee.
+    ///   - Setting it to `1` will cause the literal `#[weight = x]` values to be charged.
+    pub struct WeightToFee;
+    impl WeightToFeePolynomial for WeightToFee {
+        type Balance = Balance;
+        fn polynomial() -> WeightToFeeCoefficients<Self::Balance> {
+            // in Starlight, extrinsic base weight (smallest non-zero weight) is mapped to 1/10 CENT:
+            let p = super::currency::CENTS;
+            let q = 10 * Balance::from(ExtrinsicBaseWeight::get().ref_time());
+            smallvec![WeightToFeeCoefficient {
+                degree: 1,
+                negative: false,
+                coeff_frac: Perbill::from_rational(p % q, q),
+                coeff_integer: p / q,
+            }]
+        }
+    }
+}
+
+/// System Parachains.
+pub mod system_parachain {
+    use {primitives::Id, xcm_builder::IsChildSystemParachain};
```

Comment:
we can probably remove all these, from dancelight and starlight. we dont use this I bvelieve

Reply:
fixed